conveyor/adapter/response.go

```diff
@@ -26,8 +26,8 @@ import (
 )
 
 // NewResponseSendAdapter creates new instance of adapter for sending response
-func NewResponseSendAdapter() PulseConveyorAdapterTaskSink {
-	return NewAdapterWithQueue(NewSendResponseProcessor())
+func NewResponseSendAdapter(id uint32) TaskSink {
```

Comment:
Why not use `adapter.ID `type for this param? In that case you dont have to convert all constants to uint32